src/openvtb/components/taken/schemas.py

```diff
@@ -0,0 +1,114 @@
+from django.utils.translation import gettext as _
+
+from jsonschema import Draft202012Validator
```

Comment:
Is this the latest version? I'm not really familiar with the different validator classes from this library